packages/grafana-toolkit/src/cli/tasks/plugin.utils.ts

```diff
@@ -162,13 +162,24 @@ const githubPublishRunner: TaskRunner<GithubPublishOptions> = async ({ dryrun, v
     throw `The release plugin requires you specify the repository url as environment variable CIRCLE_REPOSITORY_URL`;
   }
 
-  if (!process.env['GITHUB_TOKEN']) {
-    throw `Github publish requires that you set the environment variable GITHUB_TOKEN to a valid github api token.
-    See: https://github.com/settings/tokens for more details.`;
+  if (!process.env['GITHUB_ACCESS_TOKEN']) {
+    // Try to use GITHUB_TOKEN, which may be set.
+    if (process.env['GITHUB_TOKEN']) {
+      process.env['GITHUB_ACCESS_TOKEN'] = process.env['GITHUB_TOKEN'];
+    } else {
+      throw `Github publish requires that you set the environment variable GITHUB_ACCESS_TOKEN to a valid github api token.
```

Comment:
We should always throw instances of `Error`. I don't know how much digging may be necessary to fix that in this PR, though.